src/dropdown/dropdown.spec.ts

```diff
@@ -131,49 +131,52 @@ describe('ngb-dropdown', () => {
     expect(dropdownEl).toHaveCssClass('dropdown');
   });
 
-  it('should allow setting a custom dropdown class (container="body")', () => {
-    const html = `
+  it('should allow setting a custom dropdown class (container="body")', fakeAsync(() => {
+       const html = `
       <div ngbDropdown placement="bottom" container="body" class="my-class" [dropdownClass]="dropdownClass">
           <button ngbDropdownAnchor></button>
           <div ngbDropdownMenu>
             <a class="dropdown-item">dropDown item</a>
           </div>
       </div>`;
 
-    // closed
-    const fixture = createTestComponent(html);
-    const dropdown = fixture.debugElement.query(By.directive(NgbDropdown)).injector.get(NgbDropdown);
+       // closed
+       const fixture = createTestComponent(html);
+       const dropdown = fixture.debugElement.query(By.directive(NgbDropdown)).injector.get(NgbDropdown);
 
-    function dropdownEl() { return document.querySelector('div[ngbDropdownMenu]') !.parentNode !; }
+       function dropdownEl() { return document.querySelector('div[ngbDropdownMenu]') !.parentNode !; }
 
-    expect(dropdownEl()).not.toHaveCssClass('custom-class');
-    expect(dropdownEl()).toHaveCssClass('my-class');
-    expect(dropdownEl()).toHaveCssClass('dropdown');
+       expect(dropdownEl()).not.toHaveCssClass('custom-class');
+       expect(dropdownEl()).toHaveCssClass('my-class');
+       expect(dropdownEl()).toHaveCssClass('dropdown');
 
-    // open
-    dropdown.open();
-    fixture.detectChanges();
+       // open
+       dropdown.open();
+       fixture.detectChanges();
+       tick(25);
```

Comment:
Seems like popper uses promises for scheduling, so 25 is not necessary, just `tick()`